modules/installation-dns-ibm-cloud.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc
+
+:_content-type: PROCEDURE
+[id="installation-dns-ibm-cloud_{context}"]
+= Using IBM Cloud DNS Services for DNS resolution
+
+The installation program uses IBM Cloud DNS Services to configure cluster DNS resolution and provide name lookup for a private cluster.
+
```

Comment:
its is the baseDomain value in install-config.yaml

Reply:
I am not sure what you mean by this. Please explain a bit more.

Reply:
yes

Reply:
Thank you. Done.